File: java-main/java-main/NestedforloopAssi/p38.java
/*
A b C d
E f G h
I j K l
M n O p
*/

class Demo{

	public static void main(String[]args){

		char ch='A';
		int N=4;

		for(int i=1;i<=N;i++){
			for(int j=1;j<=N;j++){
				System.out.print(ch++ + " ");
			}
			System.out.println();
		}
	}
}

File: bootcamp_c_cpp_dsa-main/bootcamp_c_cpp_dsa-main/CPP/codes/p38.cpp
#include<iostream>
class Demo{
	int x=10;
	int y=20;
	public:
	Demo(){
		std::cout<< "In No-args Constructor" <<std::endl;
	}
	Demo(int x,int y){
		this->x=x;
		this->y=y;
		std::cout<< "In Para Constructor" <<std::endl;
	}
	Demo(Demo &xyz){
		std::cout<< "In Copy Constructor" <<std::endl;
	}
};
int main(){
	Demo obj1;
	Demo obj2(100,200);
	Demo obj3=obj1;
	Demo obj4;
	obj4=obj1;
	return 0;
}


